addons/crm/views/crm_lead_views.xml

```diff
@@ -1095,8 +1217,69 @@ if record:
             <field name="act_window_id" ref="crm_lead_action_pipeline"/>
         </record>
 
+        <record id="action_opportunity_forecast" model="ir.actions.server">
+            <field name="name">Crm: Forecast</field>
+            <field name="model_id" ref="crm.model_crm_team"/>
+            <field name="state">code</field>
+            <field name="groups_id"  eval="[(4, ref('base.group_user'))]"/>
+            <field name="code">action = model.action_opportunity_forecast()</field>
+        </record>
+
+        <record model="ir.actions.act_window" id="crm_lead_action_forecast">
```

Comment:
Guidelines: define id before model (so that we have all ids on the left when quickly looking a file)